test/GitHub.Primitives.UnitTests/UriStringTests.cs

```diff
@@ -25,11 +25,15 @@ public class TheConstructor : TestBaseClass
         [TestCase("jane;fingerprint=9e:1a:5e:27:16:4d:2a:13:90:2c:64:41:bd:25:fd:35@foo.com:github/Windows.git",
             "foo.com", "github", "Windows")]
         [TestCase("https://haacked@bitbucket.org/haacked/test-mytest.git", "bitbucket.org", "haacked", "test-mytest")]
-        [TestCase("https://git01.codeplex.com/nuget", "git01.codeplex.com", null, "nuget")]
-        [TestCase("https://example.com/vpath/foo/bar", "example.com", "foo", "bar")]
-        [TestCase("https://example.com/vpath/foo/bar.git", "example.com", "foo", "bar")]
+        [TestCase("https://git01.codeplex.com/nuget", "git01.codeplex.com", "nuget", null,
+            Description = "We assume the first component is the owner")]
+        [TestCase("https://example.com/vpath/foo/bar", "example.com", "vpath", "foo")]
+        [TestCase("https://example.com/vpath/foo/bar.git", "example.com", "vpath", "foo")]
```

Comment:
`vpath` here suggests "virtual path" which I assume is because enterprise installations can have a virtual path before the owner? I this PR changes the parsing of this to not work with vpaths, but maybe we should change the string `vpath` to something else here?